### ISSUE TITLE
Make Insufficient Storage response to be fatal for OneDrive

### ISSUE THREAD

**Issue by Dibbyo456** [rclone/rclone#4089]
While I was syncing 50TB+ data on 5TB OneDrive remote, I noticed that Rclone keep trying to copying files one by one even though Graph API was sending HTTP status code 507 (Insufficient Storage).

Kindly make this fatal, so Rclone don't retry further.

Complete errors list: https://docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/errors?view=odsp-graph-online

**Comment by ncw** [comment]
The error looks easy to parse which is good!

Fancy haveing a go at this?

It should be a matter of adding another `case 507:` here 

https://github.com/rclone/rclone/blob/d9c8c47e028a1f90f671f7eb335e92112a56ee83/backend/onedrive/onedrive.go#L426

and returning the error wrapped in a Fatal error, something like this

```go
case 507: // Insufficient Storage
    return false, fserrors.FatalError(err)
```

**Comment by ncw** [comment]
I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.52
